paloalto/rules/dag_update_fw1.yaml

```diff
@@ -0,0 +1,21 @@
+---
+    name: "dag_update_fw1"
+    pack: "paloalto"
+    description: "update DAG for firewall 192.168.3.101 with sev filter"
+    enabled: true
+
+    trigger:
+        type: "core.st2.webhook"
+        parameters:
+            url: "paloalto"
+    
+    criteria:                              # optional
+        trigger.body.severity:
+            type: "contains"
+            pattern : "critical"       
+
+    action:
+        ref: "paloalto.dag_update"
+        parameters:
+            ip: "{{trigger.body.source}}"
+            firewall: "192.168.3.101"
```

Comment:
Since this is a public pack, why hard code the firewall address here?

Reply:
Should I leave it empty , firewall: "" ? This is meant to be example for editing